application/src/projection.rs

```diff
@@ -0,0 +1,34 @@
+use domain::todolist_event::TodoListEvent;
+use framework::*;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Clone, Default, Debug, Serialize, Deserialize)]
+pub struct TodoListProjection {
+    pub in_progress: HashMap<usize, String>,
+    pub completed: HashMap<usize, String>,
+}
```

Comment:
**Suggestion:** Consider adding a docstring to the `TodoListProjection` struct. [best practice]
```suggestion
#[derive(Clone, Default, Debug, Serialize, Deserialize)]
/// Represents the projection of the todo list.
pub struct TodoListProjection {
  pub in_progress: HashMap<usize, String>,
  pub completed: HashMap<usize, String>,
}
```